changelog/974.feature.rst

```diff
@@ -0,0 +1 @@
+Added a fitting function for 1D spectra, `~plasmapy.diagnostics.thomson.spectral_density_model`, to the Thomson scattering diagnostic module.
```

Comment:
Perhaps split the changelog into `974.feature.1.rst` for `spectral_energy_model` and  `974.feature.2.rst` for `spectral_energy_lite`?

Reply:
Good idea - done.